### PR TITLE
done

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://savchuksavchuk.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,9 +8,46 @@
...
+        <input 
+          type="text" 
+          class="form-input form-input-big"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+        <div class="form-effect"></div>
+      </form>
+      <form 
+        action="index.html"
```
Comment (alexandra-protyanova) [inline src/index.html:35]:
```suggestion
        action="/" 
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  overflow: hidden;
+  z-index: 9;
+}
+
+.form-input {
+  position: relative;
+  border: none;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, Helvetica, sans-serif;
```
Comment (alexandra-protyanova) [inline src/style.css:44]: move to `html`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+.form-img {
+  position: absolute;
+  overflow: hidden;
+  z-index: 9;
+}
+
+.form-input {
+  position: relative;
+  border: none;
+  box-sizing: border-box;
```
Comment (alexandra-protyanova) [inline src/style.css:42]: move to `*`

File: src/style.css (hunk 2 above)
Comment (savchuksavchuk) [reply src/style.css:44]: Should I use <font> tag?
There is a requirement for this task to use @font-face. But mb I don't understand something. 
Will be gratefull for a hint.

File: src/index.html [hunk 4]
```diff
@@ -8,9 +8,46 @@
...
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <main>
+      <form 
+        action="index.html"
```
[the remark this answers is not in the document]
Comment (savchuksavchuk) [reply src/index.html]: done

File: src/index.html (hunk 1 above)
Comment (savchuksavchuk) [reply src/index.html]: done

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-img {
+  position: absolute;
+  overflow: hidden;
+  z-index: 9;
```
[the remark this answers is not in the document]
Comment (savchuksavchuk) [reply src/style.css]: done

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+.form-input {
+  position: relative;
+  border: none;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-style: normal;
+  z-index: 9;
```
[the remark this answers is not in the document]
Comment (savchuksavchuk) [reply src/style.css]: done

File: src/style.css (hunk 3 above)
Comment (savchuksavchuk) [reply src/style.css]: done

File: src/style.css (hunk 2 above)
Comment (valerii-derkach) [reply src/style.css:43]: you connect styles using @font-face
use html tag for applying fonts to whole document, actually it is the requirement of @alexandra-protyanova
and it is ok to use html tag for this reason

**Review by valerii-derkach — APPROVED** [review]

**Pull request closed without merging**